scripts/resolve_container_path.py

```diff
@@ -19,24 +30,44 @@
 
 def resolve_container_path(container_path):
     container_path = os.path.realpath(container_path)
+    mountinfo = '/proc/self/mountinfo'
     try:
-        with open('/proc/self/cgroup') as f:
-            # Entries in /proc/self/cgroup look like this (note the nesting):
-            # 11:name=systemd:/docker/82c1bd2…23b5bcf/docker/6547bce…60ca5a7
-            prefix, container_id = next(f).strip().split(':')[2].split('/')[-2:]
+        with open(mountinfo) as f:
+            log.info('Found %s', mountinfo)
+            # Entries of interest in /proc/self/mountinfo look as follows:
+            # 752 744 259:2 /docker/containers/dc61d9…CONTAINER_ID…/<variable> …
+            prefix = '/docker/containers'
+            log.info('cgroup v2 prefix is %s', prefix)
+            containers = set()
+            for line in csv.reader(f, delimiter=' '):
+                path = line[3]
+                if path.startswith(prefix):
+                    log.info('Extracting container ID from %s', path)
+                    prfx, id, var = path.rsplit('/', maxsplit=2)
```

Comment:
prefix and prfx? Please!


```suggestion
                    prefix, id, var = path.rsplit('/', maxsplit=2)
```